src/posit/connect/client.py

```diff
@@ -94,6 +95,17 @@ def oauth(self) -> OAuthIntegration:
         """
         return OAuthIntegration(config=self.config, session=self.session)
 
+    @property
+    def groups(self) -> Groups:
+        """The groups resource interface.
```

Comment:
My gut response is that I'd want to see more descriptive text here, if this is used as doc text in the CLI. I don't know if "groups resource interface" will be a particularly meaningful term for our users (it isn't for me).

I see that some of this (`resource`) is terminology used throughout the package, and maybe it'll make more sense to me as I get to know it.

Reply:
Good call out. The entire Client class could use some documentation love.